test/jdk/javax/swing/JScrollBar/4865918/bug4865918.java

```diff
@@ -52,6 +56,7 @@ public static void main(String[] argv) throws Exception {
             robot.delay(1000);
 
             SwingUtilities.invokeAndWait(() -> sbar.pressMouse());
+            latch.await();
```

Comment:
You should use timed `await` and throw an exception if the wait times out. This way the timeout would be handled more gracefully… and more quickly, a two-second timeout should be enough.